corehq/apps/reports/views.py

```diff
@@ -779,7 +779,8 @@ def scheduled_report_form(self):
         web_user_emails = [
             WebUser.wrap(row['doc']).get_email()
             for row in get_all_user_rows(self.domain, include_web_users=True,
-                                         include_mobile_users=False, include_docs=True)
+                                         include_inactive=False, include_mobile_users=False,
```

Comment:
Got it, so this change is to preserve the behavior prior to the PR, since now `inactive` filtering works for web users?